src/Platform/Microsoft.Testing.Platform/Microsoft.Testing.Platform.csproj

```diff
@@ -74,6 +74,13 @@ This package provides the core platform and the .NET implementation of the proto
     <EmbeddedResource Update="Resources\PlatformResources.resx" GenerateSource="true" />
   </ItemGroup>
 
+  <ItemGroup>
+    <SupportedPlatform Include="android" />
```

Comment:
Shall we add `windows`, `linux` and `macOS` or is it default?

Reply:
They are default. https://github.com/dotnet/sdk/blob/cb459ebc2d9374b2bc5ce944cc633a6e79ed8275/src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.SupportedPlatforms.props